Feature/task 23 kazhin

Noggano94 opened the pull request.

KazhinPhilipp committed:
ПЕ-21б: Кажин Ф.А.: Запрос создания схемы и таблицы Дисциплин.

Создание схемы Discipline.
Создание таблицы Discipline.discipline.

KazhinPhilipp committed:
ПЕ-21б Кажин Ф.А.: Добавление поля инкримента в таблицу Discipline.discipline

KazhinPhilipp committed:
ПЕ-21б Кажин Ф.А.: Создание строки в таблице

Добавление в файл запроса процедуры добавления тестовой строки в таблицу Discipline.discipline

KazhinPhilipp committed:
ПЕ-12б Кажин Ф.А.: Смена кодировки запроса

KazhinPhilipp committed:
ПЕ-21б Кажин Ф.А.: Редактирование оформления SQL

Смена пробелов на табуляцию.
пробел в комментариях.
Добавление первичного ключа.

dha01 commented on SQL/Task_23/create Discipline.discipline and create schema Discipline.sql:
```diff
@@ -0,0 +1,25 @@
+	short_name varchar(30) NOT NULL,
+	full_name varchar(255) NOT NULL,
+	mem varchar(max),
+	constraint PK__task primary key (discipline),
```
Копипаст, не забываем его проверять и заменять всё на то, что нужно.
Запрос, вероятно, либо не проверялся, либо проверялся на старой базе.
constraint PK__discipline primary key (discipline)

dha01 commented on SQL/Task_23/create Discipline.discipline and create schema Discipline.sql:
```diff
@@ -0,0 +1,25 @@
+-- Создание таблицы Discipline.discipline
+create table Discipline.discipline (
+	discipline int identity(1,1),
+	short_name varchar(30) NOT NULL,
```
Регистр not null

dha01 commented on SQL/Task_23/create Discipline.discipline and create schema Discipline.sql:
```diff
@@ -0,0 +1,25 @@
+-- Создание схемы Discipline
+create schema  Discipline
```
Лишний пробел между schema и Discipline

KazhinPhilipp committed:
ПЕ-21б Кажин Ф.А.: Правильное оформление запроса.

Удален лишний пробел между schema и Discipline.
Регистр not null.
Изменение строки constraint PK__discipline primary key (discipline).

dha01 commented on IS.Mvc/IS.Mvc/IS.Mvc.csproj:
```diff
@@ -20,6 +20,11 @@
     <IISExpressAnonymousAuthentication />
     <IISExpressWindowsAuthentication />
     <IISExpressUseClassicPipelineMode />
+    <MvcProjectUpgradeChecked>true</MvcProjectUpgradeChecked>
```
Невнимательность, это не относится к задаче.

KazhinPhilipp committed:
ПЕ-21б Кажин Ф.А.: Откат к предыдущей версии случайно закомитенного файла.
